Karma/Views/Home/DonateDialog.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿
+@{
+    Layout = null;
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body>
+    <div id="donateDialog" class="modal-body">
+        <div class="container-fluid">
+            <div class="row">
+                @*TODO: redirect to page with person requests*@
+                <div class="col-6 btn-image">
```

Comment:
is this built in css class? col-6 btn-image

Reply:
col-6 is Bootstrap, btn-image is my custom class defined in site.css